lib/Chat/ReactionManager.php

```diff
@@ -119,6 +121,7 @@ public function deleteReactionMessage(Room $chat, string $actorType, string $act
 			])
 		);
 		$comment->setVerb(ChatManager::VERB_REACTION_DELETED);
+		$comment->setExpireDate($parentComment->getExpireDate());
```

Comment:
I fear this should be in the addSystemMessage
Now you are changing the expiration data of the original message.

But I guess that one does it already.

Reply:
Why should the reaction deleted comment not expire if the parent comment expires?